lib/appeals/configuration.rb

```diff
@@ -32,7 +32,7 @@ def connection
     end
 
     def mock_enabled?
-      Settings.appeals.mock
+      [true, 'true'].include?(Settings.appeals.mock)
```

Comment:
I think `Settings.appeals.mock || false` is more readable.  Granted, doesn't catch a string `'true'`.